src/backend/src/routes/tasks.routes.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import { body } from 'express-validator';
+import TasksController from '../controllers/tasks.controllers';
+import { isTaskStatus, isTaskPriority } from '../utils/tasks.utils';
+import { validateInputs } from '../utils/utils';
+import { intMinZero } from '../utils/validation.utils';
+
+const tasksRouter = express.Router();
+
+tasksRouter.post(
+  '/',
+  intMinZero(body('wbsElementId')),
+  body('title').isString(),
+  body('notes').isString(),
+  body('deadline').isDate(),
+  isTaskPriority(body('priority')),
+  isTaskStatus(body('status')),
+  body('assignees').isArray(),
```

Comment:
check that it's array of numbers

Reply:
uhhh can we specify whats inside the array? we dont do that anywhere else in codebase

Reply:
yes, and we definitely do

it's like `body('assignees.*').isInt() min 0 blah blah